pkg/tsdb/prometheus/json_testdata_create.go

```diff
@@ -0,0 +1,58 @@
+package prometheus
```

Comment:
i'm not 100% happy with the filename here, also wanted to put it into a separate package, but there was an import cycle happening, because "this package" was importing `github.com/grafana/grafana/pkg/tsdb/prometheus` to get access to `prometheus.PrometheusQuery`, and `github.com/grafana/grafana/pkg/tsdb/prometheus` package was also importing this file, so i left it in the same package.

if you have an idea how to solve this, or recommend a better file name, i'll gladly change this.

Reply:
It's fine to live in this package because it's alongside the code under test.